### PR TITLE
nodeDriverRegistrar.healthPort, affinity, and more tolerations

### PR DESCRIPTION
**Pull request by KenHuffmanAtNice**
Replace remaining 8081 with server.port.

Signed-off-by: Ken Huffman ken.huffman@niceincontact.com

### CODE REVIEW

**Comment by marcofranssen** [comment]
Could you sign the DCO and fix the failed build?

Please rebase on top of main to resolve the conflicts.

File: charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml [hunk 1]
```diff
@@ -79,7 +87,7 @@ spec:
             - name: kubelet-plugin-registration-dir
               mountPath: /registration
           ports:
-            - containerPort: 9809
+            - containerPort: {{ .Values.nodeDriverRegistrar.healthPort }}
```
Comment (marcofranssen) [inline charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml:90]: why make this configurable. Just adds to the complexity of the chart, where probably no one ever has to change this.

File: charts/spire/charts/spire-server/templates/statefulset.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
             - /run/spire/config/server.conf
           ports:
             - name: grpc
-              containerPort: 8081
+              containerPort: {{ .Values.service.port }}
```
Comment (marcofranssen) [inline charts/spire/charts/spire-server/templates/statefulset.yaml:44]: doesn't have to necesarrely be the same port as mapped in the service.

File: charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml (hunk 1 above)
Comment (KenHuffmanAtNice) [reply charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml:90]: This port unfortunately conflicts with AWS fargate on our clusters, and has to be changed for our usage.

File: charts/spire/charts/spire-server/templates/statefulset.yaml (hunk 2 above)
Comment (KenHuffmanAtNice) [reply charts/spire/charts/spire-server/templates/statefulset.yaml:44]: If this wasn't an oversight/bug, I will revert this change.

**Pull request closed without merging**